### PR TITLE
SAK-49350 roster. Remove User object from serialised JSON

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-49350

### CODE REVIEW

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 1]
```diff
@@ -7063,6 +7063,8 @@ private void post_save_submission(RunData data, boolean post) {
      * Takes the inline submission, prepares it as an attachment to the submission and queues the attachment with the content review service
      */
     private void prepareInlineForContentReview(String text, AssignmentSubmission submission, SessionState state, User student) {
+
+        System.out.println("HERE1");
```
Comment (jesusmmp) [inline assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:7067]: Please, remove this. Thanks!

Comment (adrianfish) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:7067]: good spot, thanks. From another ticket :)

**Comment by jesusmmp** [comment]
Thanks @adrianfish!
Let me test this patch in our instance :)

**Review by jesusmmp — APPROVED** [review]
Just tested! Great job.

Thanks @adrianfish!

**Comment by adrianfish** [comment]
Thanks for the testing @jesusmmp

**Pull request merged**